Gemfile.lock

```diff
@@ -1,8 +1,8 @@
 GEM
```

Comment:
GitHub Codespaces use Ruby 3 - These changes were required to get things working